tests/test_tti.py

```diff
@@ -93,8 +94,8 @@ def ricker_source(t, f0):
                       .5 * u_tti.data.reshape(-1) - .5 * v_tti.data.reshape(-1))
     res /= linalg.norm(u.data.reshape(-1))
     log("Difference between acoustic and TTI with all coefficients to 0 %f" % res)
-    assert np.isclose(res, 0.0, atol=1e-1)
+    assert np.isclose(res, 0.0, atol=1e-4)
```

Comment:
really?

Reply:
Yes,centred is more accurate for weak (small) anisotropy

Reply:
centered is what we had before?

Reply:
Shifted is what we had before